### PR TITLE
Remove old structure level functionality

### PR DESCRIPTION
**Pull request by jsangmeister**
sets up on https://github.com/OpenSlides/openslides-backend/pull/1989

part of https://github.com/OpenSlides/openslides-backend/issues/1948

TODO: add migration

### CODE REVIEW

File: openslides_backend/action/actions/user/update.py [hunk 1]
```diff
@@ -47,7 +47,6 @@ class UserUpdate(
             "gender",
             "email",
             "default_number",
-            "default_structure_level",
```
Comment (r-peschke) [inline openslides_backend/action/actions/user/update.py:50]: Why didn't you add structure_level_ids? You did in create_update_permissions.

File: openslides_backend/action/actions/user/create.py [hunk 2]
```diff
@@ -44,7 +44,6 @@ class UserCreate(
             "gender",
             "email",
             "default_number",
-            "default_structure_level",
```
Comment (r-peschke) [inline openslides_backend/action/actions/user/create.py:47]: Why didn't you add structure_level_ids? You did in create_update_permissions.

**Review by r-peschke — COMMENTED** [review]
See remarks on user.update/create.
Also you should write some notes in action wiki including the hint for the feature branch (or not?). This concerns the account/participant.json_upload, and the user actions.

File: openslides_backend/action/actions/user/update.py (hunk 1 above)
Comment (jsangmeister) [reply openslides_backend/action/actions/user/update.py:50]: `default_structure_level` was removed completely without replacement. `structure_level` was replaced with `structure_level_ids`, which are present a few lines below in the `UserMixin.transfer_list`.

**Review by r-peschke — COMMENTED** [review]
Please change the action wiki concerning user.create/update, account- and participant.json_upload
Will you add the migration to this PR or write a new one?

**Comment by jsangmeister** [comment]
> Please change the action wiki concerning user.create/update, account- and participant.json_upload

Sure, what do you want added there? Just a hint that the imports will need changes after this is merged?

> Will you add the migration to this PR or write a new one?

A new one, see https://github.com/OpenSlides/openslides-backend/pull/2000

**Comment by r-peschke** [comment]
> > Please change the action wiki concerning user.create/update, account- and participant.json_upload
> 
> Sure, what do you want added there? Just a hint that the imports will need changes after this is merged?
> 
You could mark the old fields as deprecated with the new branch and the new ones with the feature branch name, which will introduce them. On merging we just had to remove the comments.

> > Will you add the migration to this PR or write a new one?
> 
> A new one, see #2000

**Comment by jsangmeister** [comment]
> You could mark the old fields as deprecated with the new branch and the new ones with the feature branch name, which will introduce them. On merging we just had to remove the comments.

I updated `user.{create|update}` and `account.json_upload` with a hint just like the other structure level actions. Since `participant.json_upload` should stay the same from a payload perspectuve

**Comment by jsangmeister** [comment]
Merged in https://github.com/OpenSlides/openslides-backend/pull/2000

**Pull request closed without merging**